backend/src/components/edx/exchange.js

```diff
@@ -238,6 +238,29 @@ async function markAs(req, res) {
   }
 }
 
+async function markAsClosed(req, res) {
+  const token = utils.getBackendToken(req);
+  if (!token) {
+    return res.status(HttpStatus.UNAUTHORIZED).json({
+      message: 'No access token'
+    });
+  }
+  try {
+    const currentExchange = await getData(token, config.get('server:edx:exchangeURL') + `/${req.params.secureExchangeID}`);
+    currentExchange.secureExchangeStatusCode = 'CLOSED';
+    currentExchange.createDate = null;
+    currentExchange.updateDate = null;
+
```

Comment:
Please add a check here to confirm the user has the `SECURE_EXCHANGE` role and correct team access to make the change. Check the exchange for the ownership team and confirm the user has the access.

Reply:
Please have another look. I have implemented permission validation requested.